llif/exec.go

```diff
@@ -0,0 +1,20 @@
+package llif
+
+type ExecGenericInput struct {
+	// The state of LL handlers
+	FSState      *LLState
+	NetworkState *LLState
```

Comment:
not really clear why the exec input struct needs network and fs state. Same for the fs and network input structs.

Reply:
The main reason is because the input parameters usually depend on some of the FS state. For example, for executing the nabla container, we need the file path of the ISO that was created during the FS phase.

Reply:
OK, i see. So, there is some input common to everybody? don't want to complicate things too much, but would it be a good idea to extract that common input, so that all the each subsystem gets only relevant stuff: its input and the common input?

Reply:
Actually, would execState be that common input?

Reply:
Yea you are absolutely right. We should scale it down to what is needed. I plan to get some feedback from everyone about this. Because i'm not sure what else everyone needs. Like in our case for network we only need the container ID.